client/src/components/application/SelectBox/index.js

```diff
@@ -6,51 +6,51 @@ import Select from '@material-ui/core/Select';
 import PropTypes from 'prop-types';
 import './style.css';
 
-export default class SelectBox extends React.Component {
-  state = {
-    selectValue: '',
-  };
+const SelectBox = ({
+  label,
+  items,
+  className,
+  onChange,
+  name,
+  selectValue,
+}) => (
+  <div>
+    <FormControl variant="outlined" fullWidth>
+      <InputLabel className="select-box" id="demo-simple-select-outlined-label">
+        {label}
+      </InputLabel>
```

Comment:
Ahmed did many edits on select box, I advice you to see his edits also, if we accepted his pull request first, then you have to do some changes here also .